cores/esp8266/Esp.cpp

```diff
@@ -203,6 +204,15 @@ void EspClass::restart(void)
     esp_yield();
 }
 
+void EspClass::reboot_uart_dwnld()
```

Comment:
The class use `camelCase` for other methods, can we do the same here?  And no need to shorten names, our assembler doesn't have a limited identifier len. :)  `rebootUartDownload()` or something?